unikernel.ml

```diff
@@ -754,12 +754,26 @@ struct
                    reply);
              [])
         >>= fun unikernels ->
+        (Albatross.query albatross ~domain:u.name (`Policy_cmd `Policy_info)
```

Comment:
I am a bit unsure about the policies and our approach for the source of truth... so we have albatross running somewhere, do we expect (and support) that some other means (e.g. command-line) is used for inserting/adding/removing policies?

Or should mollymawk be run under the assumption that only mollymawk is used to maintain albatross? So we can retrieve and cache the policies at startup and are sure they'll not change over time.

As said, I'm undecided - it is mainly a question about how we expect mollymawk to be used...